logType.js

```diff
@@ -0,0 +1,29 @@
+const data = ;
+switch(typeof data) {
+case 'undefined':
+	console.log("This is a undefined!");
+	break;
+case 'null':
+	console.log("This is null!");
+	break;
+case 'number':
+	console.log(isNaN(data) ? "This is not a number!" : "This is a number!");
+	break;
+case 'string':
+	console.log("This is a string!");
+	break;
+case 'boolean':
+	console.log("This is a boolean!");
+	break;
+case 'bigint':
+	console.log("This is a bigint!");
+	break;
+case 'function':
+	console.log("This is a function!");
+	break;
+case 'object':
+		console.log(data === null ? "This is null!" : (Array.isArray(data) ? "This is an array!" : "This is an object!"));
```

Comment:
it looks like it works, Sven suggested only using the ternary operator with one condition and not to nest them. writing the same thing as else if would make it easier to read